packages/Popover/src/Popover.js

```diff
@@ -438,14 +445,22 @@ class Popover extends React.Component {
       <ThemeContext.Provider value={isDark}>
         <PopoverContext.Provider value={contextValue}>
           <PopoverStyled data-pka-anchor="popover" {...moreProps} ref={this.$popover}>
-            {this.props.children}
+            <PopoverChildren onChildChange={this.handleChildChange}>{this.props.children}</PopoverChildren>
           </PopoverStyled>
         </PopoverContext.Provider>
       </ThemeContext.Provider>
     );
   }
 }
 
+// Todo Refactor this when we convert popover component to use hooks
+function PopoverChildren(props) {
+  React.useLayoutEffect(() => {
+    if (props.children) props.onChildChange();
+  }, [props.children]);
```

Comment:
So I think I discussed this with @mikrotron?.. but because `children` are usually react elements, which are JS objects, this will basically trigger any time the popover's parent re-renders. We can probably go with this for now and see whether this causes issues in practice.

Reply:
Ok yeah not sure of best solution to prevent that at this point, but se can see what issues arise.

Reply:
Yeah, let's see. Popovers are closed most of the time, and when they're open, the user is likely focussed on them and not causing re-renders, so it seems like this won't come up too often anyway?